src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java

```diff
@@ -165,7 +166,25 @@ private void changeProject(final Project newProject, final long minusSeconds) {
       if (hideable) {
          mainStage.hide();
       }
+      if (model.emptyNoteReminder.get()) {
+         final Work currentWork = model.activeWorkItem.get();
+         if (currentWork != null && currentWork.getNotes().isEmpty()) {
+            final Alert alert = new Alert(AlertType.CONFIRMATION);
```

Comment:
Please adapt the dialog so the user can directly add a note. More userfriendly. This was a 'feature' in a eariler version which was removed as there was no setting yet. See: https://github.com/doubleSlashde/KeepTime/blob/master/src/main/java/de/doubleslash/keeptime/viewpopup/ViewControllerPopup.java#L65
Input field should be focused so the user can directly type and press enter to confirm the switch (with our without added note :))

Reply:
Please add the current project name to the dialog :) Similar to how it is in the linked code. May forget what project is currently active :D